packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -243,6 +277,8 @@ const ProductDetail = () => {
         }
     }, [product])
 
+    const normalizedProduct = normalizeSetBundleProduct(product)
```

Comment:
Minor: how about `comboProduct` to be clearer? Instead of normalized, I feel that we want to emphasize that this is a combination product (either a set or a bundle).